### PR TITLE
[fix](inverted index) implementation of match_regexp function without index

### PR DESCRIPTION
**Pull request by zzzxl1993**
… indexing

## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by zzzxl1993** [comment]
run buildall

**Comment by qidaye** [comment]
run buildall

File: be/src/vec/functions/match.cpp [hunk 1]
```diff
@@ -393,6 +393,91 @@ Status FunctionMatchPhrasePrefix::execute_match(
...
+                                          size_t input_rows_count, const ColumnString* string_col,
+                                          InvertedIndexCtx* inverted_index_ctx,
+                                          const ColumnArray::Offsets64* array_offsets,
+                                          ColumnUInt8::Container& result) const {
+    DBUG_EXECUTE_IF("match.invert_index_not_support_execute_match", {
+        return Status::Error<ErrorCode::INVERTED_INDEX_NOT_SUPPORTED>(
+                "FunctionMatchRegexp not support execute_match");
+    })
+
+    LOG(ERROR) << "FunctionMatchRegexp execute_match";
```
Comment (qidaye) [inline be/src/vec/functions/match.cpp:407]: Remove this log

**Comment by zzzxl1993** [comment]
run buildall

**Review by qidaye — DISMISSED** [review]
LGTM

**Comment by zzzxl1993** [comment]
run buildall

File: be/src/vec/functions/match.cpp [hunk 2]
```diff
@@ -393,6 +395,89 @@ Status FunctionMatchPhrasePrefix::execute_match(
...
+    auto on_match = [](unsigned int id, unsigned long long from, unsigned long long to,
+                       unsigned int flags, void* context) -> int {
+        *((bool*)context) = true;
+        return 0;
+    };
+
+    try {
+        auto current_src_array_offset = 0;
+        for (int i = 0; i < input_rows_count; i++) {
+            std::vector<std::string> data_tokens =
```
Comment (airborne12) [inline be/src/vec/functions/match.cpp:454]: auto & data_tokens

**Review by airborne12 — APPROVED** [review]
LGTM

**Comment by zzzxl1993** [comment]
run performance

**Review by qidaye — APPROVED** [review]

**Pull request merged**